app/src/main/res/layout-v23/selected_card_layout.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
actually you aren't gonna need `RelativeLayout` here. You can safely remove it and use `ConstrainLayout` as root layout